test/Microsoft.SqlTools.ServiceLayer.Test/Connection/ConnectionServiceTests.cs

```diff
@@ -61,29 +57,26 @@ public DbConnection CreateMockDbConnection(Dictionary<string, string>[][] data)
         [Fact]
         public void CanCancelConnectRequest()
         {
-            var testFile = "file:///my/test/file.sql";
+            const string testFile = "file:///my/test/file.sql";
```

Comment:
Agreed. I have some other changes I'd like to make to properly isolate and share commonly used code in the test suites. It'll be in a forthcoming PR.